be/src/formats/parquet/file_reader.cpp

```diff
@@ -135,17 +136,21 @@ Status FileReader::_parse_footer(FileMetaData** file_metadata, size_t* metadata_
     RETURN_IF_ERROR(deserialize_thrift_msg(reinterpret_cast<const uint8*>(footer_buffer.data()) + footer_buffer.size() -
                                                    PARQUET_FOOTER_SIZE - metadata_length,
                                            &metadata_length, TProtocolType::COMPACT, &t_metadata));
-    *file_metadata = new FileMetaData();
-    *metadata_size = metadata_length;
-    RETURN_IF_ERROR((*file_metadata)->init(t_metadata, _scanner_ctx->case_sensitive));
+    int64_t before_bytes = CurrentThread::current().get_consumed_bytes();
+    {
```

Comment:
why do we need code block here

Reply:
> why do we need code block here

I intended to represent a time range by it, but it seems unnecessary. Changed.